io.github.realmazharhussain.GdmSettings.yaml

```diff
@@ -10,7 +10,6 @@ finish-args:
   - --socket=wayland
   - --socket=fallback-x11
   - --talk-name=org.freedesktop.Flatpak
-  - --filesystem=xdg-config/monitors.xml:ro
```

Comment:
Hi, is this right? Seems natural that a GDM manager wants to access host's monitof config?

Reply:
Flathub doesn't allow that anymore.

Reply:
Yes please ask for an exception, the exception system is in place for exactly this.

Reply:
I did not know there was such a system before #37. Now, I have asked for an exception. https://github.com/flathub-infra/flatpak-builder-lint/pull/317